tests/datasource/fluent/test_batch.py

```diff
@@ -156,3 +160,46 @@ def test_batch_validate_change_expectation_suite_result_format(
     assert boolean_result.success is True
     assert len(boolean_result.results) == 1
     assert len(boolean_result.results[0].result) == 0
+
+
+@pytest.mark.filesystem
```

Comment:
These tests were nice to have for the red/green cycle, but I'm not sure that tests asserting a lack of side effects are terribly meaningful. Reviewer: WDYT?

Reply:
since they're already written, i think its probably worthwhile to keep them - they do capture behavior we intend, and helps guard against someone accidentally changing the code to persist these in the future